lib/pbench/server/api/__init__.py

```diff
@@ -305,9 +326,23 @@ def post(self):
             app.logger.exception("There was something wrong uploading {}", filename)
             abort(500, message=f"There was something wrong uploading {filename}")
 
-        if hash_md5.hexdigest() != md5sum:
-            abort(400, message=f"md5sum check failed for {filename}")
+        message = None
+        if bytes_received > content_length:
+            message = f"More bytes received than the specified content length {content_length}"
```

Comment:
We also want to complain if we get too _few_ bytes.

Reply:
Ahh right